examples/vector-labels-justify-text.js

```diff
@@ -59,7 +59,7 @@ const features = [
   },
 ];
 
-function createStyle({textAlign, justify}) {
```

Comment:
Object unpacking requires properties to be set, but `justify` is an optional property.